topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTest.java

```diff
@@ -177,6 +177,16 @@ public void testOnKafkaTopicAdded_invalidResource(VertxTestContext context) {
 
                 assertThat(registry.get(TopicOperator.METRICS_PREFIX + "reconciliations.duration").tag("kind", "KafkaTopic").timer().count(), is(0L));
                 assertThat(registry.get(TopicOperator.METRICS_PREFIX + "reconciliations.duration").tag("kind", "KafkaTopic").timer().totalTime(TimeUnit.MILLISECONDS), is(0.0));
+
+                assertThat(registry.get(TopicOperator.METRICS_PREFIX + "resource.state")
+                        .tag("kind", "KafkaTopic")
+                        .tag("name", "invalid")
+                        .tag("resource-namespace", "default-namespace")
+                        .tag("reason", "KafkaTopic's spec.config has invalid entry: The key 'null' of the topic config is invalid: The value corresponding to the key must have a string, number or boolean value but the value was null")
```

Comment:
You do use this string multiple times. Maybe it would be worth it to save it into the variable.